base/loading.jl

```diff
@@ -2505,26 +2505,33 @@ function check_clone_targets(clone_targets)
     end
 end
 
+const OPT_LEVEL = 5
```

Comment:
Why is this a global?

Reply:
I wanted to mimick https://github.com/JuliaLang/julia/pull/48179/files#diff-1adc5c0f4e183651111760a5b2fdaf21418217de8142791cf1214709ae49dbc6R615

but yeah this isn't scoped.